src/clj/game/cards-resources.clj

```diff
@@ -759,6 +759,11 @@
                                :no-ability {:effect (effect (clear-wait-prompt :corp))}}}
                             card nil))}}}
 
+   "Tech Trader"
+   {:events {:runner-trash {:req (req (and (= side :runner) (= (second targets) :ability-cost)))
+                            :msg "Gain a credit"
```

Comment:
Remember that `:msg` is always shown as "PlayerName uses CardName to ... [msg]", so this should be `"gain 1 [Credit]"`.